be/src/main/java/codesquad/issuetracker/domain/enumtype/IssueStatus.java

```diff
@@ -0,0 +1,5 @@
+package codesquad.issuetracker.domain.enumtype;
```

Comment:
enum 자체로 열거형 타입을 나타내고 있다고 생각됩니다 :)
```suggestion
package codesquad.issuetracker.domain.enum;
```

Reply:
enum을 packing하는데 practice를 찾던 중 아래 스택오버플로우 글을 발견했는데,

enum만 따로 패키지화하는데는 별다른 이점이 없다는 내용이었습니다. 나름대로 enum을 별도 패키지로 묶는 것이 낫다고 생각했었지만 이러한 답변이 그래도 꽤나 지지를 받고 있는 것으로 보여 별도 패키징을 하지 않기로 했습니다. 😂 (김영한님 JPA 강의에서도 별도로 enum을 패키지로 묶지는 않았습니다. -> 참고용) 

※ 참고자료 : https://stackoverflow.com/questions/36220718/is-there-a-java-convention-for-packaging-enums

Reply:
enumtype 이라는 패키지로 이넘을 분류하신것 아닌가요...?